hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -3,18 +3,1146 @@
+             derived clocks.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_sw_clk_off_peri
+      desc: '''Verify the ability to turn off the peripheral clock via SW.
+
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_clk_div
+      desc: '''Verify clk division logic is working correctly.
+
+
+            '''
+      milestone: V2
+      tests: []
+    }
+
+    // PWRMGR tests:
+
+    // RSTMGR tests:
+
+    // ALERT_HANDLER (pre-verified IP) integration tests:
+    {
+      name: chip_alert_handler_alerts
+      desc: '''Verify all alerts coming into the alert_handler.
+
+            X-ref'ed with all IP tests.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_alert_handler_irqs
+      desc: '''Verify all classes of alert handler interrupts to the CPU.
+
+
+            Program each alert to cause an interrupt in each class. SW validates the reception of
+            the interrupt. X-ref'ed with all IP tests.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_alert_handler_esc_irqs
+      desc: '''Verify all alert handler escalation irqs.
+
+
+            Details TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_alert_handler_entropy
+      desc: '''Verify all alert handler entropy input to ensure pseudo-random ping timer.
+
+
+            Details TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_alert_handler_crashdump
+      desc: '''Verify the alert handler crashdump signal.
+
+
+            Details TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+
+    // LC_MANAGER (pre-verified IP) integration tests:
+
+
+    ///////////////////////////////////////////////////////
+    // Security Peripherals                              //
+    // AES, HMAC, KMAC, CSRNG, ENTROPY_SRC, KEYMGR, OTBN //
+    ///////////////////////////////////////////////////////
+
+    // AES (pre-verified IP) integration tests:
+    {
+      name: chip_aes_enc
+      desc: '''Verify the AES operation.
+
+            Write a 32-byte key and a 16-byte plain text to the AES registers and trigger the AES
+            computation to start. Wait for the AES operation to complete by polling the status
+            register (or interrupt if available). Check the digest registers for correctness against
+            the expected digest value.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_aes_shadow_reg_alert
+      desc: '''Verify shadow reg alert from AES.
+
+            Inject a storage error via backdoor to generate this alert signal while the SW is
+            actively executing some piece of code. Verify alert propagation to an NMI.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
```

Comment:
i know you know this already.. but just to make a note.
I think MOST LIKELY this interface will not exist.

Reply:
Ok, thanks for confirming. I have removed it.